src/utils/ValidationHelper.ts

```diff
@@ -102,27 +102,85 @@ export class ValidationHelper {
 		return "";
 	};
 
-	isAddressFormatValid = (jwtPayload: JwtPayload): { errorMessage: string; errorMessageCode: string } => {
-		const personIdentityAddresses: PersonIdentityAddress[] = jwtPayload.shared_claims.address;
-		for (const address of personIdentityAddresses) {
-			if (!this.checkIfValidCountryCode(address.addressCountry)) {
-				return {
-					errorMessage: "Invalid country code: country code is not GB in the postalAddress",
-					errorMessageCode: MessageCodes.INVALID_COUNTRY_CODE,
-				};
-			} else if (!this.checkIfAddressIsValid(address)) {
-				// Validation fails if all the mandatory postalAddress fields- subBuildingName, buildingName, buildingNumber and streetName are missing and is not a valid string or if all the 3 mandatory fields- subBuildingName, buildingName, buildingNumber are missing or not a valid string
-				return {
-					errorMessage: "Missing all or some of mandatory postalAddress fields (subBuildingName, buildingName, buildingNumber and streetName), unable to create the session",
-					errorMessageCode: MessageCodes.MISSING_ALL_MANDATORY_POSTAL_ADDRESS_FIELDS,
-				};
-			}
+	isSharedClaimDataValid(jwtPayload: JwtPayload): { errorMsg: string; errorMsgCode: string } {
```

Comment:
[Nit] for consistency could we please use `errorMessage` and `errorMessageCode`?

Reply:
Not sure what happened, I had pushed the PR comments code changes in, but dont see them here :(. Have pushed them now. Kindly review.